ideascube/serveradmin/catalog.py

```diff
@@ -346,6 +361,15 @@ class StaticSite(SimpleZipPackage):
     typename = 'static-site'
     handler = Nginx
 
+    @property
+    def template_id(self):
+         if self.id == 'w2eu':
+             return self.id
+         template_id, extension = self.id.rsplit('.', 1)
+         if extension == 'map':
+             template_id = 'maps'
+         return template_id
```

Comment:
So many special cases. :disappointed: 

Isn't it easier to rename the card template file names?